### DISCUSSION TITLE
Will upcoming release v7 of sequelize won't support oracle as it is missing in official documentation of alpha v7?

### DISCUSSION THREAD

**Discussion by kom24al** [sequelize/sequelize#17239] [Help & Questions] [answered]
Sequelize v7 (alpha)
npm version npm downloads sponsor Last commit Merged PRs GitHub stars node License

Sequelize is a promise-based [Node.js](https://nodejs.org/en/about/) [ORM tool](https://en.wikipedia.org/wiki/Object-relational_mapping) for [Postgres](https://en.wikipedia.org/wiki/PostgreSQL), [MySQL](https://en.wikipedia.org/wiki/MySQL), [MariaDB](https://en.wikipedia.org/wiki/MariaDB), [SQLite](https://en.wikipedia.org/wiki/SQLite), [Microsoft SQL Server](https://en.wikipedia.org/wiki/Microsoft_SQL_Server), [Amazon Redshift](https://docs.aws.amazon.com/redshift/index.html) and [Snowflake’s Data Cloud](https://docs.snowflake.com/en/user-guide/intro-key-concepts.html). It features solid transaction support, relations, eager and lazy loading, read replication and more.

**Comment by WikiRik** [comment, marked as answer]
Hi! We're working together with Oracle to make sure we have support when the first stable release of v7 comes out. Until that support is added, Oracle is only supported on v6
